lib/validation.js

```diff
@@ -19,10 +19,6 @@
  * @module selectValidation
  */
 
-module.exports = function selectValidation(input, settings) {
-  if (input.target.value === '' && !settings.target.empty) {
-    return `${input.target.name} cannot be left blank!`;
-  }
-
+module.exports = function selectValidation() {
```

Comment:
Should there be validation here? If the selects are related, should there be some validation enforcing their relationship?